site/_posts/2023-02-03-single-sign-on-laravel-fusionauth.md

```diff
@@ -0,0 +1,339 @@
+
+<!--more-->
+
+The trust is established through an out-of-band exchange of cryptographic credentials, such as a client secret or public key infrastructure (PKI) certificate. Once trust has been established between the service provider and the identity provider, SSO authentication can occur when a user wants to authenticate with the service provider. This will typically involve the service provider sending a token to the identity provider containing details about the user seeking authentication. The identity provider can then check if the user is already authenticated and ask them to authenticate if they are not. Once this is done, the identity provider can send a token back to the service provider, signifying that the user has been granted access.
+
+SSO helps reduce password fatigue by requiring users to only remember one password and username for all the applications managed through the SSO feature. This also reduces the number of support tickets created for your IT team when a user inevitably forgets their password. In addition, SSO minimizes the number of times you have to key-in security credentials, limiting exposure to security issues like keystroke probing and exposure. Security is also enhanced by SSO because you can implement additional functionality such as MFA or anomalous behavior detection at the identity provider without adding any complexity to your application.
+
+In this tutorial, you'll learn how to design and implement SSO using [Laravel](https://laravel.com), a popular PHP-based web framework and FusionAuth as the OIDC provider. Any other OIDC compatible authentication server should work as well.
+
+## Implementing SSO in a Laravel web app
+
+As previously stated, in this tutorial, you'll be shown how to implement SSO in a Laravel web app. The Laravel demo application is integrated with FusionAuth, an authentication and authorization platform, and [Laravel Socialite](https://laravel.com/docs/9.x/socialite), its official solution to authenticate users with OAuth providers.
+
+Before you begin, you'll need the following:
+
+* A Linux machine. The step-by-step instructions in this article are based on a [CentOS Linux machine](https://www.centos.org). If you want to work on a different OS, check out this [setup guide](https://fusionauth.io/docs/v1/tech/5-minute-setup-guide) for more information.
+* [Docker Engine](https://docs.docker.com/engine/) and [Docker Compose](https://docs.docker.com/compose/).
+* Experience with a Laravel framework and application development.
+* The [Git client tool](https://git-scm.com/downloads).
+
+## Clone the GitHub repository
+
+To begin, clone the demo project source code for this project from this [GitHub repository](https://github.com/FusionAuth/fusionauth-example-laravel-single-sign-on) using the following command:
```

Comment:
Why do we need to do this? Tell me why, please. (Seems like it provides some useful migrations/code that can be copied into an existing project.)

Reply:
I've rephrased that